### ISSUE TITLE
Add completed nandgame save?

### ISSUE THREAD

**Issue by SlothScript** [timlg07/TheNandGame-solutions#15]
Nandgame got updated so if you could provide a save.json file with a completed nandgame save that would be helpful

**Comment by timlg07** [comment]
true, thanks for pointing out the import/export functionality. I will play it again and provide the solution as JSON.

**Comment by Tommic7** [comment]
is it done yet????

**Comment by timlg07** [comment]
> is it done yet????

Unfortunately, I didn't have time for it yet 💀 But I will add it to my to-do, so hopefully soon ^^
